python/paddle/fluid/dygraph/dygraph_to_static/variable_trans_func.py

```diff
@@ -34,6 +34,28 @@ def create_static_variable_gast_node(name):
     return gast.parse(func_code)
 
 
+def create_fill_constant_node(name, value):
+    func_code = "{} = fluid.layers.fill_constant(shape=[1], ".format(name)
+    if isinstance(value, bool):
+        func_code += "dtype='bool', value={})".format(value)
+        return gast.parse(func_code)
+    if isinstance(value, float):
+        func_code += "dtype='float64', value={})".format(value)
+        return gast.parse(func_code)
+
+    if six.PY2:
+        if isinstance(value, int):
+            func_code += "dtype='int32', value={})".format(value)
+            return gast.parse(func_code)
+        if isinstance(value, long):
+            func_code += "dtype='int64', value={})".format(value)
+            return gast.parse(func_code)
+    else:
+        if isinstance(value, int):
+            func_code += "dtype='int64', value={})".format(value)
+            return gast.parse(func_code)
+
```

Comment:
Is the expected return value of `create_fill_constant_node ` is `gast.Assigin` ?  
The type of `gast.parse(func_code)` is `gast.Module` and the type of `gast.parse(func_code).body[0]` is `gast.Assigin`

Reply:
Thank you for clarifying. Although these code passes tests and still workable as gast.Module, I think changing them as you suggested is better